cmd/lambda/accounts/update_test.go

```diff
@@ -1,325 +1,129 @@
 package main
 
 import (
-	"context"
-	"errors"
 	"fmt"
-	commonMocks "github.com/Optum/dce/pkg/common/mocks"
-	"github.com/Optum/dce/pkg/db"
-	util "github.com/Optum/dce/tests/testutils"
-	"github.com/aws/aws-lambda-go/events"
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/service/sts"
-	"github.com/stretchr/testify/mock"
-	"github.com/stretchr/testify/require"
+	"io/ioutil"
+	"net/http/httptest"
+	"strings"
 	"testing"
+	"time"
+
+	amMocks "github.com/Optum/dce/pkg/accountmanager/accountmanageriface/mocks"
+	"github.com/Optum/dce/pkg/config"
+	dataMocks "github.com/Optum/dce/pkg/data/dataiface/mocks"
+	"github.com/Optum/dce/pkg/model"
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
 )
 
-func TestUpdateAccountByID(t *testing.T) {
```

Comment:
@kddejong -- what's the story here? There's behavior captured in these tests that I'm not seeing in the new test coverage, and it's not clear if the refactored controller implements the behavior.

I'd like to avoid breaking changes here if we can...

Reply:
I think you'll need to be more specific.  Some of that logic may have moved someplace else.

Reply:
talked offline about this -- I was asking specifically about test coverage around which fields are update-able and which aren't. @kddejong says this is covered under the validation logic in `account.Update()`